### PR TITLE
1597 create tests for npc specific prefabs inside class prefabc

### LINKED ISSUE #1597: Create tests for NPC-specific prefabs inside class_prefab.c
Jonathan, Vincent, and I wrote friendly npc-specific prefabs inside the playerclass module. All of our code has compiled with the six new classes we created (alchemist, chef, healer, shopkeeper, fisherman, lord). We now plan to write tests specific to these new classes in tests/playerclass/test_class_prefabs.c to ensure that all of the new descriptions and stats are suiting for the current chiventure structure. We also want to make sure that the code can still compile and doesn't interfere. The changes we made to class_prefab.c is detailed in issue #1490. This issue is dedicated to writing tests for the code written in #1490.

### PR DESCRIPTION
**Pull request by katie-gao**
Wrote a few tests for the npc specific prefabs in tests/test_class_prefabs.c. This is related to issue #1490, where we wrote npc specific prefabs (alchemist, chef, healer, shopkeeper, fisherman, lord), and wanted to test out the code. This is a really small change in the testing file, and as there are no skills for these classes - not as much to test.

### CODE REVIEW

File: src/playerclass/src/class_prefabs.c [hunk 1]
```diff
@@ -200,8 +223,9 @@ class_t* class_prefab_new(game_t* game, char* class_name) {
      * 10 Max Mana */
     else if (!strncmp(temp_name, "ranger", MAX_NAME_LEN)) {
         short_desc = "A master hunter.";
-        long_desc = "The ranger is the embodiment of an apex predator: while they may tend to lurk away "
-                    "from civilisation in the wild, they are a skilled killer and have no qualms about doing so.";
+        long_desc = "The ranger is the embodiment of an apex predator: while they may "
+                    " tend to lurk away from civilisation in the wild, they are a "
```
Comment (wellingtj) [inline src/playerclass/src/class_prefabs.c]: there are two spaces here

File: src/playerclass/src/class_prefabs.c [hunk 2]
```diff
@@ -252,6 +277,109 @@ class_t* class_prefab_new(game_t* game, char* class_name) {
         set_stats_hashtable(game, &stats, 10, 10, 5, 5, 10, 20, 25, 25);
     }
 
+    /*-----------------------------------------------------------------
+     *-------------------Player-specific prefabs-----------------------
```
Comment (wellingtj) [inline src/playerclass/src/class_prefabs.c]: npc specific?

**Review by wellingtj — APPROVED** [review]
Looks good to me. I left a few minor comments but they're very quick fixes. Overall great code

File: src/playerclass/src/class_prefabs.c (hunk 1 above)
Comment (jroven) [reply src/playerclass/src/class_prefabs.c]: fixed

File: src/playerclass/src/class_prefabs.c (hunk 2 above)
Comment (jroven) [reply src/playerclass/src/class_prefabs.c]: fixed

**Review by lilyehsani — APPROVED** [review]
LGTM! Approved.

**Comment by lilyehsani** [comment]
Issue Score: Excellent

Comments:
Great work! Like issues, PRs should have closing statements.

**Pull request merged**